<!-- -->
<launch>

	<arg name="linear_interpolate_mode" default="True"/>
	<!-- rosrun waypoint_follower pure_pursuit -->
	<node pkg="waypoint_follower" type="pure_pursuit" name="pure_pursuit" output="log">
		<param name="linear_interpolate_mode" value="$(arg linear_interpolate_mode)"/>
        <param name="publish_frequency" value="50" />
        <param name="subscriber_queue_length" value="1" />
        <param name="const_lookahead_distance" value="3.0" />
        <param name="minimum_lookahead_distance" value="3.0" />
        <param name="lookahead_distance_ratio" value="4.0" />
        <param name="maximum_lookahead_distance_ratio" value="15" />
	</node>

</launch>
